packages/frontend-shared/cypress/support/mock-graphql/mountFragment.ts

```diff
@@ -48,6 +48,7 @@ export const registerMountFn = ({ plugins }: MountFnOptions = {}) => {
         },
       })
 
+      // @ts-ignore todo: figure out the correct types
```

Comment:
I messed with the type errors in this file for a while... still not sure what needs to change. We already ignore some types in this file (see line 27)